docs/docsite/rst/administration/instances.rst

```diff
@@ -234,6 +236,99 @@ You can remove an instance by clicking **Remove** in the Instances page, or by s
+
+.. image:: ../common/images/instances_meshingress_listener_addresses.png
+		:alt: Details of each listener address associated with the mesh ingress.
+
+In the corresponding Topology view, you can see the AWX ingress node connected to the control plane.
+
+.. image:: ../common/images/topology-view-with-ingress.png
+	:alt: Topology view with ingress node connected to the control plane.
+
+
+6. Create a virtual machine, an instance that only allows inbound traffic for port 22 for SSH and that no other inbound traffic is allowed to this VM. This VM will be peering to the AWX control plane. An AWS VM is used for this example.
+
+7. Create an external execution node instance (leave listener port blank because the VM spun up doesn't allow any inbound traffic) that is capable of running playbooks (the ec2 node). See :ref:`ag_instances_add` for detail.
+
+8. Once the instance is created, peer the execution node to the mesh ingress node (awx-mesh-ingress-1).
+
+.. image:: ../common/images/instances-peer-execution-with-ingress.png
+	:alt: Associating a peer window showing a arrow pointing at the mesh ingress node.
+
+9. Download the install bundle from the instance's Details view (|download|) and install the requirements:
+
+::
+
+	install -r requirements.yml
+
+.. |download| image:: ../common/images/download-icon.png
+	:alt: Download icon.
+
+
+10. Run the playbook, specifying the ansible user and password that is needed to connect to this instance. This example playbook requires sudo permission.
+
+::
+
+	ansible-playbook install_receptor.yml -e ansible_user=ec2-user -I inventory.yml -v
+
+11. After the playbook finishes running, the topology view now shows the AWX control plane and the execution node peering into the mesh ingress node.
+
+.. image:: ../common/images/instances_peerings_into-mesh-ingress-node.png
+	:alt: Topology view showing execution node and control plane peering into the mesh ingress node.
+
+
+12. Run a health check against the execution node to ensure the execution node is successfully added and ready to be added to an instance group.
+
+.. image:: ../common/images/instances-execution-node-status-ready.png
+.. image:: ../common/images/topology-view-execution-node-status-ready.png
+	:alt: Execution node showing ready status in the Details view and in the Topology view.
+
+
+13. Create an instance group and add the execution node to it in order for it to be used in your job execution. Whatever execution environment image used to run a playbook needs to be accessible for your remote execution node. Everything you are using in your playbook also needs to be accessible from this remote execution node.
+
+.. image:: ../common/images/instances-job-template-using-remote-execution-ig.png
+	:alt: Job template using the instance group with the execution node to run jobs.
+
+Launching the job template will take a while in order to pull down new images. 
+
+
 Using a custom Receptor CA
```

Comment:
Remove the contents and use cross reference top operator-docs on this: https://ansible.readthedocs.io/projects/awx-operator/en/stable/user-guide/advanced-configuration/adding-execution-nodes.html#custom-receptor-ca